mne/viz/_3d.py

```diff
@@ -1395,3 +1396,205 @@ def _get_view_to_display_matrix(scene):
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions. It corresponds to
+        Freesurfer environment variable SUBJECTS_DIR. If None (default),
+        SUBJECTS_DIR is read from environment or config file.
+    coord_frame : str
+        Coordinate frame to use, 'head' or 'mri'.
+    idx : int | 'gof' | 'amplitude'
+        Index of the initially plotted dipole. Can also be 'gof' to plot the
+        dipole with highest goodness of fit value or 'amplitude' to plot the
+        dipole with the highest amplitude. The dipoles can also be browsed
+        through using up/down arrow keys or mouse scroll. Defaults to 'gof'.
+    ax : instance of matplotlib Axes3D | None
+        Axes to plot into. If None (default), axes will be created.
+    block : bool
+        Whether to halt program execution until the figure is closed. Defaults
+        to False.
+    show : bool
+        Show figure if True. Defaults to True.
+
+    Returns
+    -------
+    fig : instance of matplotlib figure
+        The figure containing 3-D locations of the dipoles.
+    """
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.mplot3d import Axes3D
+    if has_nibabel():
+        import nibabel as nib
+        from nibabel.processing import resample_from_to
+    else:
+        raise ImportError('This function requires nibabel.')
+    if coord_frame not in ['head', 'mri']:
+        raise ValueError("coord_frame must be 'head' or 'mri'. "
+                         "Got %s." % coord_frame)
+    if idx == 'gof':
+        idx = np.argmax(dipole.gof)
+    elif idx == 'amplitude':
+        idx = np.argmax(dipole.amplitude)
+    elif not isinstance(idx, int):
+        raise ValueError("idx must be an int or one of ['gof', 'amplitude']. "
+                         "Got %s." % idx)
+    subjects_dir = get_subjects_dir(subjects_dir=subjects_dir,
+                                    raise_error=True)
+    t1_fname = op.join(subjects_dir, subject, 'mri', 'T1.mgz')
+    t1 = nib.load(t1_fname)
+    ras2vox = np.linalg.inv(t1.header.get_vox2ras_tkr())
+    trans = _get_trans(trans, fro='head', to='mri')[0]
+    if coord_frame == 'head':
+        affine_to = trans['trans'].copy()
+        affine_to[:3, 3] *= 1000  # to mm
+        aff = t1.affine.copy()
+        zooms = t1.header.get_zooms()
+        aff[:3, :3] /= zooms
+        affine_to = np.dot(affine_to, aff)
+        t1 = resample_from_to(t1, ([int(t1.shape[i] * zooms[i]) for i
+                                    in range(3)], affine_to))
+    else:
+        zooms = (1., 1., 1.)
+
+    data = t1.get_data()
+    dims = len(data)  # Symmetric size assumed.
+
+    fig = plt.figure()
+    ax = Axes3D(fig) if ax is None else ax
+    if coord_frame == 'head':
+        dipole_locs = dipole.pos
+        ori = dipole.ori
+        dd = (dims / 2.) / 1000.
+        gridx, gridy = np.meshgrid(np.linspace(-dd, dd, dims),
+                                   np.linspace(-dd, dd, dims))
```

Comment:
I'm not sure I understand the question. Doesn't it depend on the coordinate frame by definition (eg. -0.1 - 0.1 vs. 0 - 256)?